### DISCUSSION TITLE
Random assignment of participants

### DISCUSSION THREAD

**Discussion by ler93** [jspsych/jsPsych#2140] [Q&A] [answered]
Hi there! I'm using cognition.run for an experiment that includes 4 conditions. I don't know how to make the program randomly assign participants to different conditions. The only modification that will be established for each one will be a reading text. The rest of the experiment will be the same for everyone. I hope you can help me! Thanks in advance.

**Comment by GEJ1** [comment, marked as answer]
Hi @ler93 ,

If you go to configuration > advance configurations there is a named **_Inter experiment conditions_** that says:  

> Used in between-group (or between-groups) study design. Automatic assignment of participants to different conditions. If you specify a number of condition greater than 1, then Cognition will attempt to assign new participants to conditions to keep them all with equal N. The assigned condition will be accessible during experiment runtime using the variable CONDITION

Hope it helps!

best,

Gustavo.

**Reply by ler93** [reply]
Thanks for your answer, @GEJ1. Yes, I had seen the option you mention. I think that I didn't formulate my previous question very well. What I'm trying to figure out is how to use the CONDITION variable. Is there any link of reference that explains it? Thanks again!

**Reply by GEJ1** [reply]
Oh sorry, I misunderstood. I'm not sure if there is more documentation about cognition.run functionalities.

**Reply by jodeleeuw** [reply]
Check out this answer in another discussion: https://github.com/jspsych/jsPsych/discussions/754#discussioncomment-827860

**Reply by ler93** [reply]
Thank you so much @jodeleeuw! I found an easy way to do it from that link.
